[08-26 14:01:41]: [INFO] Created column 'regdate' and set the current timestamp, 1756216901360, to all 0 rows
[08-26 14:01:41]: [INFO] SQLite Setup finished
[08-26 14:01:41]: [INFO] No supported permissions system found! Permissions are disabled!
[08-26 14:01:42]: [WARN] WARNING! The protectInventory feature requires ProtocolLib! Disabling it...
[08-26 14:01:43]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[08-26 14:01:49]: [INFO] Downloading GEO IP database, because the old database is older than 30 days or doesn't exist
[08-26 14:01:49]: [WARN] No MaxMind credentials found in the configuration file! GeoIp protections will be disabled.
[08-26 14:01:49]: [INFO] There is no newer GEO IP database uploaded to MaxMind. Using the old one for now.
[08-26 14:01:49]: [WARN] Could not download GeoLiteAPI database [FileNotFoundException]: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
[08-26 14:01:49]: java.io.FileNotFoundException: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
	at java.base/java.io.RandomAccessFile.open0(Native Method)
	at java.base/java.io.RandomAccessFile.open(RandomAccessFile.java:356)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:273)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:223)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.BufferHolder.<init>(BufferHolder.java:20)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.Reader.<init>(Reader.java:121)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.startReading(GeoIpService.java:179)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.updateDatabase(GeoIpService.java:150)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftTask.run(CraftTask.java:101)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[08-26 14:08:30]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[08-26 14:12:05]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[08-26 14:12:08]: [WARN] Can't initialize the MySQL database: [SQLException]: Access denied for user 'TemSaborDMel'@'localhost' (using password: YES)
[08-26 14:12:08]: java.sql.SQLException: Access denied for user 'TemSaborDMel'@'localhost' (using password: YES)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:130)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:815)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:438)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:241)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:189)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:561)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:112)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.MySQL.getConnection(MySQL.java:185)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.MySQL.checkTablesAndColumns(MySQL.java:193)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.MySQL.<init>(MySQL.java:78)
	at AuthMe-5.6.0.jar//fr.xephi.authme.initialization.DataSourceProvider.createDataSource(DataSourceProvider.java:68)
	at AuthMe-5.6.0.jar//fr.xephi.authme.initialization.DataSourceProvider.get(DataSourceProvider.java:50)
	at AuthMe-5.6.0.jar//fr.xephi.authme.initialization.DataSourceProvider.get(DataSourceProvider.java:26)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.injector.handlers.instantiation.ProviderHandler$InstantiationByProviderClass.instantiateWith(ProviderHandler.java:129)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.injector.InjectorImpl.resolveContext(InjectorImpl.java:164)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.injector.InjectorImpl.resolve(InjectorImpl.java:133)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.injector.InjectorImpl.getSingleton(InjectorImpl.java:72)
	at AuthMe-5.6.0.jar//fr.xephi.authme.AuthMe.instantiateServices(AuthMe.java:245)
	at AuthMe-5.6.0.jar//fr.xephi.authme.AuthMe.initialize(AuthMe.java:221)
	at AuthMe-5.6.0.jar//fr.xephi.authme.AuthMe.onEnable(AuthMe.java:145)
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:281)
	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:189)
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104)
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507)
	at org.bukkit.craftbukkit.v1_19_R3.CraftServer.enablePlugin(CraftServer.java:563)
	at org.bukkit.craftbukkit.v1_19_R3.CraftServer.enablePlugins(CraftServer.java:474)
	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:638)
	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:437)
	at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:308)
	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1104)
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[08-26 14:12:08]: [WARN] Please check your database settings in the config.yml file!
[08-26 14:12:08]: [WARN] Could not create data source: [SQLException]: Access denied for user 'TemSaborDMel'@'localhost' (using password: YES)
[08-26 14:12:08]: java.sql.SQLException: Access denied for user 'TemSaborDMel'@'localhost' (using password: YES)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:130)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:815)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:438)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:241)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:189)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:561)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:112)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.MySQL.getConnection(MySQL.java:185)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.MySQL.checkTablesAndColumns(MySQL.java:193)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.MySQL.<init>(MySQL.java:78)
	at AuthMe-5.6.0.jar//fr.xephi.authme.initialization.DataSourceProvider.createDataSource(DataSourceProvider.java:68)
	at AuthMe-5.6.0.jar//fr.xephi.authme.initialization.DataSourceProvider.get(DataSourceProvider.java:50)
	at AuthMe-5.6.0.jar//fr.xephi.authme.initialization.DataSourceProvider.get(DataSourceProvider.java:26)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.injector.handlers.instantiation.ProviderHandler$InstantiationByProviderClass.instantiateWith(ProviderHandler.java:129)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.injector.InjectorImpl.resolveContext(InjectorImpl.java:164)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.injector.InjectorImpl.resolve(InjectorImpl.java:133)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.injector.InjectorImpl.getSingleton(InjectorImpl.java:72)
	at AuthMe-5.6.0.jar//fr.xephi.authme.AuthMe.instantiateServices(AuthMe.java:245)
	at AuthMe-5.6.0.jar//fr.xephi.authme.AuthMe.initialize(AuthMe.java:221)
	at AuthMe-5.6.0.jar//fr.xephi.authme.AuthMe.onEnable(AuthMe.java:145)
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:281)
	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:189)
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104)
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507)
	at org.bukkit.craftbukkit.v1_19_R3.CraftServer.enablePlugin(CraftServer.java:563)
	at org.bukkit.craftbukkit.v1_19_R3.CraftServer.enablePlugins(CraftServer.java:474)
	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:638)
	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:437)
	at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:308)
	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1104)
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[08-26 14:12:08]: [WARN] Aborting initialization of AuthMe: [IllegalStateException]: Error during initialization of data source
[08-26 14:12:08]: java.lang.IllegalStateException: Error during initialization of data source
	at AuthMe-5.6.0.jar//fr.xephi.authme.initialization.DataSourceProvider.get(DataSourceProvider.java:53)
	at AuthMe-5.6.0.jar//fr.xephi.authme.initialization.DataSourceProvider.get(DataSourceProvider.java:26)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.injector.handlers.instantiation.ProviderHandler$InstantiationByProviderClass.instantiateWith(ProviderHandler.java:129)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.injector.InjectorImpl.resolveContext(InjectorImpl.java:164)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.injector.InjectorImpl.resolve(InjectorImpl.java:133)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.ch.jalu.injector.InjectorImpl.getSingleton(InjectorImpl.java:72)
	at AuthMe-5.6.0.jar//fr.xephi.authme.AuthMe.instantiateServices(AuthMe.java:245)
	at AuthMe-5.6.0.jar//fr.xephi.authme.AuthMe.initialize(AuthMe.java:221)
	at AuthMe-5.6.0.jar//fr.xephi.authme.AuthMe.onEnable(AuthMe.java:145)
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:281)
	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:189)
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104)
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507)
	at org.bukkit.craftbukkit.v1_19_R3.CraftServer.enablePlugin(CraftServer.java:563)
	at org.bukkit.craftbukkit.v1_19_R3.CraftServer.enablePlugins(CraftServer.java:474)
	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:638)
	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:437)
	at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:308)
	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1104)
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.sql.SQLException: Access denied for user 'TemSaborDMel'@'localhost' (using password: YES)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:130)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:815)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:438)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:241)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:189)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:364)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:476)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:561)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:112)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.MySQL.getConnection(MySQL.java:185)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.MySQL.checkTablesAndColumns(MySQL.java:193)
	at AuthMe-5.6.0.jar//fr.xephi.authme.datasource.MySQL.<init>(MySQL.java:78)
	at AuthMe-5.6.0.jar//fr.xephi.authme.initialization.DataSourceProvider.createDataSource(DataSourceProvider.java:68)
	at AuthMe-5.6.0.jar//fr.xephi.authme.initialization.DataSourceProvider.get(DataSourceProvider.java:50)
	... 20 more

[08-26 14:12:08]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[08-26 14:21:19]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[08-26 14:21:22]: [INFO] MySQL setup finished
[08-26 14:21:22]: [INFO] No supported permissions system found! Permissions are disabled!
[08-26 14:21:22]: [WARN] WARNING! The protectInventory feature requires ProtocolLib! Disabling it...
[08-26 14:21:23]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[08-26 14:21:29]: [INFO] Downloading GEO IP database, because the old database is older than 30 days or doesn't exist
[08-26 14:21:29]: [WARN] No MaxMind credentials found in the configuration file! GeoIp protections will be disabled.
[08-26 14:21:29]: [INFO] There is no newer GEO IP database uploaded to MaxMind. Using the old one for now.
[08-26 14:21:29]: [WARN] Could not download GeoLiteAPI database [FileNotFoundException]: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
[08-26 14:21:29]: java.io.FileNotFoundException: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
	at java.base/java.io.RandomAccessFile.open0(Native Method)
	at java.base/java.io.RandomAccessFile.open(RandomAccessFile.java:356)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:273)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:223)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.BufferHolder.<init>(BufferHolder.java:20)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.Reader.<init>(Reader.java:121)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.startReading(GeoIpService.java:179)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.updateDatabase(GeoIpService.java:150)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftTask.run(CraftTask.java:101)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[08-26 14:24:17]: [FINE] TemSaborDeMel registered 179.186.211.154
[08-26 14:24:17]: [FINE] TemSaborDeMel logged in 179.186.211.154
[08-26 15:56:10]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[08-26 15:56:13]: [INFO] MySQL setup finished
[08-26 15:56:13]: [INFO] No supported permissions system found! Permissions are disabled!
[08-26 15:56:14]: [WARN] WARNING! The protectInventory feature requires ProtocolLib! Disabling it...
[08-26 15:56:15]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[08-26 15:56:20]: [INFO] Downloading GEO IP database, because the old database is older than 30 days or doesn't exist
[08-26 15:56:20]: [WARN] No MaxMind credentials found in the configuration file! GeoIp protections will be disabled.
[08-26 15:56:20]: [INFO] There is no newer GEO IP database uploaded to MaxMind. Using the old one for now.
[08-26 15:56:20]: [WARN] Could not download GeoLiteAPI database [FileNotFoundException]: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
[08-26 15:56:20]: java.io.FileNotFoundException: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
	at java.base/java.io.RandomAccessFile.open0(Native Method)
	at java.base/java.io.RandomAccessFile.open(RandomAccessFile.java:356)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:273)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:223)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.BufferHolder.<init>(BufferHolder.java:20)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.Reader.<init>(Reader.java:121)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.startReading(GeoIpService.java:179)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.updateDatabase(GeoIpService.java:150)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftTask.run(CraftTask.java:101)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[08-26 15:57:10]: [FINE] TemSaborDeMel used the wrong password
[08-26 15:57:21]: [FINE] TemSaborDeMel used the wrong password
[08-26 15:57:37]: [FINE] TemSaborDeMel used the wrong password
[08-26 16:13:38]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[08-26 16:14:37]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[08-26 16:14:40]: [INFO] MySQL setup finished
[08-26 16:14:40]: [INFO] No supported permissions system found! Permissions are disabled!
[08-26 16:14:40]: [WARN] WARNING! The protectInventory feature requires ProtocolLib! Disabling it...
[08-26 16:14:41]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[08-26 16:14:46]: [INFO] Downloading GEO IP database, because the old database is older than 30 days or doesn't exist
[08-26 16:14:46]: [WARN] No MaxMind credentials found in the configuration file! GeoIp protections will be disabled.
[08-26 16:14:46]: [INFO] There is no newer GEO IP database uploaded to MaxMind. Using the old one for now.
[08-26 16:14:46]: [WARN] Could not download GeoLiteAPI database [FileNotFoundException]: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
[08-26 16:14:46]: java.io.FileNotFoundException: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
	at java.base/java.io.RandomAccessFile.open0(Native Method)
	at java.base/java.io.RandomAccessFile.open(RandomAccessFile.java:356)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:273)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:223)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.BufferHolder.<init>(BufferHolder.java:20)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.Reader.<init>(Reader.java:121)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.startReading(GeoIpService.java:179)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.updateDatabase(GeoIpService.java:150)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftTask.run(CraftTask.java:101)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[08-26 16:15:27]: [FINE] TemSaborDeMel used the wrong password
[08-26 16:15:39]: [FINE] TemSaborDeMel used the wrong password
[08-26 16:16:39]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[08-26 16:18:31]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[08-26 16:18:33]: [INFO] MySQL setup finished
[08-26 16:18:34]: [INFO] No supported permissions system found! Permissions are disabled!
[08-26 16:18:34]: [WARN] WARNING! The protectInventory feature requires ProtocolLib! Disabling it...
[08-26 16:18:35]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[08-26 16:18:40]: [INFO] Downloading GEO IP database, because the old database is older than 30 days or doesn't exist
[08-26 16:18:40]: [WARN] No MaxMind credentials found in the configuration file! GeoIp protections will be disabled.
[08-26 16:18:40]: [INFO] There is no newer GEO IP database uploaded to MaxMind. Using the old one for now.
[08-26 16:18:40]: [WARN] Could not download GeoLiteAPI database [FileNotFoundException]: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
[08-26 16:18:40]: java.io.FileNotFoundException: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
	at java.base/java.io.RandomAccessFile.open0(Native Method)
	at java.base/java.io.RandomAccessFile.open(RandomAccessFile.java:356)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:273)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:223)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.BufferHolder.<init>(BufferHolder.java:20)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.Reader.<init>(Reader.java:121)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.startReading(GeoIpService.java:179)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.updateDatabase(GeoIpService.java:150)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftTask.run(CraftTask.java:101)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[08-26 16:18:54]: [FINE] TemSaborDeMel used the wrong password
[08-26 16:19:42]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[08-26 16:20:19]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[08-26 16:20:22]: [INFO] MySQL setup finished
[08-26 16:20:22]: [INFO] No supported permissions system found! Permissions are disabled!
[08-26 16:20:22]: [WARN] WARNING! The protectInventory feature requires ProtocolLib! Disabling it...
[08-26 16:20:23]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[08-26 16:20:28]: [INFO] Downloading GEO IP database, because the old database is older than 30 days or doesn't exist
[08-26 16:20:28]: [WARN] No MaxMind credentials found in the configuration file! GeoIp protections will be disabled.
[08-26 16:20:28]: [INFO] There is no newer GEO IP database uploaded to MaxMind. Using the old one for now.
[08-26 16:20:28]: [WARN] Could not download GeoLiteAPI database [FileNotFoundException]: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
[08-26 16:20:28]: java.io.FileNotFoundException: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
	at java.base/java.io.RandomAccessFile.open0(Native Method)
	at java.base/java.io.RandomAccessFile.open(RandomAccessFile.java:356)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:273)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:223)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.BufferHolder.<init>(BufferHolder.java:20)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.Reader.<init>(Reader.java:121)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.startReading(GeoIpService.java:179)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.updateDatabase(GeoIpService.java:150)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftTask.run(CraftTask.java:101)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[08-26 16:20:48]: [FINE] TemSaborDeMel used the wrong password
[08-26 16:31:55]: [FINE] TemSaborDeMel used the wrong password
[08-26 16:34:03]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[08-26 16:34:30]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[08-26 16:34:32]: [INFO] MySQL setup finished
[08-26 16:34:32]: [INFO] No supported permissions system found! Permissions are disabled!
[08-26 16:34:33]: [WARN] WARNING! The protectInventory feature requires ProtocolLib! Disabling it...
[08-26 16:34:34]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[08-26 16:34:39]: [INFO] Downloading GEO IP database, because the old database is older than 30 days or doesn't exist
[08-26 16:34:39]: [WARN] No MaxMind credentials found in the configuration file! GeoIp protections will be disabled.
[08-26 16:34:39]: [INFO] There is no newer GEO IP database uploaded to MaxMind. Using the old one for now.
[08-26 16:34:39]: [WARN] Could not download GeoLiteAPI database [FileNotFoundException]: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
[08-26 16:34:39]: java.io.FileNotFoundException: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
	at java.base/java.io.RandomAccessFile.open0(Native Method)
	at java.base/java.io.RandomAccessFile.open(RandomAccessFile.java:356)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:273)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:223)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.BufferHolder.<init>(BufferHolder.java:20)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.Reader.<init>(Reader.java:121)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.startReading(GeoIpService.java:179)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.updateDatabase(GeoIpService.java:150)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftTask.run(CraftTask.java:101)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[08-26 16:35:28]: [FINE] TemSaborDeMel used the wrong password
[08-26 16:35:38]: [FINE] TemSaborDeMel used the wrong password
[08-26 16:35:51]: [FINE] TemSaborDeMel used the wrong password
[08-26 16:36:14]: [FINE] TemSaborDeMel used the wrong password
[08-26 16:36:25]: [FINE] TemSaborDeMel used the wrong password
[08-26 16:38:40]: [FINE] TemSaborDeMel used the wrong password
[08-26 16:45:14]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[08-26 16:45:51]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[08-26 16:45:54]: [INFO] MySQL setup finished
[08-26 16:45:54]: [INFO] No supported permissions system found! Permissions are disabled!
[08-26 16:45:55]: [WARN] WARNING! The protectInventory feature requires ProtocolLib! Disabling it...
[08-26 16:45:55]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[08-26 16:46:00]: [INFO] Downloading GEO IP database, because the old database is older than 30 days or doesn't exist
[08-26 16:46:00]: [WARN] No MaxMind credentials found in the configuration file! GeoIp protections will be disabled.
[08-26 16:46:00]: [INFO] There is no newer GEO IP database uploaded to MaxMind. Using the old one for now.
[08-26 16:46:00]: [WARN] Could not download GeoLiteAPI database [FileNotFoundException]: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
[08-26 16:46:00]: java.io.FileNotFoundException: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
	at java.base/java.io.RandomAccessFile.open0(Native Method)
	at java.base/java.io.RandomAccessFile.open(RandomAccessFile.java:356)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:273)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:223)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.BufferHolder.<init>(BufferHolder.java:20)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.Reader.<init>(Reader.java:121)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.startReading(GeoIpService.java:179)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.updateDatabase(GeoIpService.java:150)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftTask.run(CraftTask.java:101)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[08-26 16:53:49]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[08-26 16:54:19]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[08-26 16:54:21]: [INFO] MySQL setup finished
[08-26 16:54:22]: [INFO] No supported permissions system found! Permissions are disabled!
[08-26 16:54:22]: [WARN] WARNING! The protectInventory feature requires ProtocolLib! Disabling it...
[08-26 16:54:23]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[08-26 16:54:28]: [INFO] Downloading GEO IP database, because the old database is older than 30 days or doesn't exist
[08-26 16:54:28]: [WARN] No MaxMind credentials found in the configuration file! GeoIp protections will be disabled.
[08-26 16:54:28]: [INFO] There is no newer GEO IP database uploaded to MaxMind. Using the old one for now.
[08-26 16:54:28]: [WARN] Could not download GeoLiteAPI database [FileNotFoundException]: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
[08-26 16:54:28]: java.io.FileNotFoundException: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
	at java.base/java.io.RandomAccessFile.open0(Native Method)
	at java.base/java.io.RandomAccessFile.open(RandomAccessFile.java:356)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:273)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:223)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.BufferHolder.<init>(BufferHolder.java:20)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.Reader.<init>(Reader.java:121)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.startReading(GeoIpService.java:179)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.updateDatabase(GeoIpService.java:150)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftTask.run(CraftTask.java:101)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[08-26 16:54:50]: [FINE] TemSaborDeMel used the wrong password
[08-26 17:12:28]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[08-26 17:18:23]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[08-26 17:18:26]: [INFO] MySQL setup finished
[08-26 17:18:26]: [INFO] No supported permissions system found! Permissions are disabled!
[08-26 17:18:26]: [WARN] WARNING! The protectInventory feature requires ProtocolLib! Disabling it...
[08-26 17:18:27]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[08-26 17:18:32]: [INFO] Downloading GEO IP database, because the old database is older than 30 days or doesn't exist
[08-26 17:18:32]: [WARN] No MaxMind credentials found in the configuration file! GeoIp protections will be disabled.
[08-26 17:18:32]: [INFO] There is no newer GEO IP database uploaded to MaxMind. Using the old one for now.
[08-26 17:18:32]: [WARN] Could not download GeoLiteAPI database [FileNotFoundException]: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
[08-26 17:18:32]: java.io.FileNotFoundException: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
	at java.base/java.io.RandomAccessFile.open0(Native Method)
	at java.base/java.io.RandomAccessFile.open(RandomAccessFile.java:356)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:273)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:223)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.BufferHolder.<init>(BufferHolder.java:20)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.Reader.<init>(Reader.java:121)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.startReading(GeoIpService.java:179)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.updateDatabase(GeoIpService.java:150)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftTask.run(CraftTask.java:101)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[08-26 17:26:25]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[08-26 17:26:58]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[08-26 17:27:01]: [INFO] MySQL setup finished
[08-26 17:27:02]: [INFO] No supported permissions system found! Permissions are disabled!
[08-26 17:27:02]: [WARN] WARNING! The protectInventory feature requires ProtocolLib! Disabling it...
[08-26 17:27:03]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[08-26 17:27:09]: [INFO] Downloading GEO IP database, because the old database is older than 30 days or doesn't exist
[08-26 17:27:09]: [WARN] No MaxMind credentials found in the configuration file! GeoIp protections will be disabled.
[08-26 17:27:09]: [INFO] There is no newer GEO IP database uploaded to MaxMind. Using the old one for now.
[08-26 17:27:09]: [WARN] Could not download GeoLiteAPI database [FileNotFoundException]: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
[08-26 17:27:09]: java.io.FileNotFoundException: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
	at java.base/java.io.RandomAccessFile.open0(Native Method)
	at java.base/java.io.RandomAccessFile.open(RandomAccessFile.java:356)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:273)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:223)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.BufferHolder.<init>(BufferHolder.java:20)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.Reader.<init>(Reader.java:121)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.startReading(GeoIpService.java:179)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.updateDatabase(GeoIpService.java:150)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftTask.run(CraftTask.java:101)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[08-26 17:27:52]: [FINE] TemSaborDeMel registered 179.186.211.154
[08-26 17:27:52]: [FINE] TemSaborDeMel logged in 179.186.211.154
[08-26 17:28:15]: [FINE] TemSaborDeMel logged in 179.186.211.154
[08-26 22:02:47]: [FINE] TemSaborDeMel logged in 179.186.211.154
[08-26 22:05:08]: [FINE] TemSaborDeMel logged in 179.186.211.154
[08-26 22:10:02]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[08-26 22:10:41]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[08-26 22:10:44]: [INFO] MySQL setup finished
[08-26 22:10:44]: [INFO] No supported permissions system found! Permissions are disabled!
[08-26 22:10:44]: [WARN] WARNING! The protectInventory feature requires ProtocolLib! Disabling it...
[08-26 22:10:45]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[08-26 22:10:51]: [INFO] Downloading GEO IP database, because the old database is older than 30 days or doesn't exist
[08-26 22:10:51]: [WARN] No MaxMind credentials found in the configuration file! GeoIp protections will be disabled.
[08-26 22:10:51]: [INFO] There is no newer GEO IP database uploaded to MaxMind. Using the old one for now.
[08-26 22:10:51]: [WARN] Could not download GeoLiteAPI database [FileNotFoundException]: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
[08-26 22:10:51]: java.io.FileNotFoundException: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
	at java.base/java.io.RandomAccessFile.open0(Native Method)
	at java.base/java.io.RandomAccessFile.open(RandomAccessFile.java:356)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:273)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:223)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.BufferHolder.<init>(BufferHolder.java:20)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.Reader.<init>(Reader.java:121)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.startReading(GeoIpService.java:179)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.updateDatabase(GeoIpService.java:150)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftTask.run(CraftTask.java:101)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[08-26 22:10:56]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[08-26 22:15:10]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[08-26 22:15:13]: [INFO] MySQL setup finished
[08-26 22:15:13]: [INFO] No supported permissions system found! Permissions are disabled!
[08-26 22:15:14]: [WARN] WARNING! The protectInventory feature requires ProtocolLib! Disabling it...
[08-26 22:15:14]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[08-26 22:15:23]: [INFO] Downloading GEO IP database, because the old database is older than 30 days or doesn't exist
[08-26 22:15:23]: [WARN] No MaxMind credentials found in the configuration file! GeoIp protections will be disabled.
[08-26 22:15:23]: [INFO] There is no newer GEO IP database uploaded to MaxMind. Using the old one for now.
[08-26 22:15:23]: [WARN] Could not download GeoLiteAPI database [FileNotFoundException]: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
[08-26 22:15:23]: java.io.FileNotFoundException: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
	at java.base/java.io.RandomAccessFile.open0(Native Method)
	at java.base/java.io.RandomAccessFile.open(RandomAccessFile.java:356)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:273)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:223)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.BufferHolder.<init>(BufferHolder.java:20)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.Reader.<init>(Reader.java:121)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.startReading(GeoIpService.java:179)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.updateDatabase(GeoIpService.java:150)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftTask.run(CraftTask.java:101)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[08-26 22:19:08]: [FINE] TemSaborDeMel logged in 179.186.211.154
[08-26 22:30:21]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[08-26 22:36:39]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[08-26 22:36:41]: [INFO] MySQL setup finished
[08-26 22:36:42]: [INFO] No supported permissions system found! Permissions are disabled!
[08-26 22:36:42]: [WARN] WARNING! The protectInventory feature requires ProtocolLib! Disabling it...
[08-26 22:36:43]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[08-26 22:36:49]: [INFO] Downloading GEO IP database, because the old database is older than 30 days or doesn't exist
[08-26 22:36:49]: [WARN] No MaxMind credentials found in the configuration file! GeoIp protections will be disabled.
[08-26 22:36:49]: [INFO] There is no newer GEO IP database uploaded to MaxMind. Using the old one for now.
[08-26 22:36:49]: [WARN] Could not download GeoLiteAPI database [FileNotFoundException]: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
[08-26 22:36:49]: java.io.FileNotFoundException: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
	at java.base/java.io.RandomAccessFile.open0(Native Method)
	at java.base/java.io.RandomAccessFile.open(RandomAccessFile.java:356)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:273)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:223)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.BufferHolder.<init>(BufferHolder.java:20)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.Reader.<init>(Reader.java:121)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.startReading(GeoIpService.java:179)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.updateDatabase(GeoIpService.java:150)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftTask.run(CraftTask.java:101)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[08-26 22:37:04]: [FINE] TemSaborDeMel logged in 179.186.211.154
[08-27 01:17:12]: [FINE] TemSaborDeMel logged in 179.186.211.154
[08-27 01:50:44]: [FINE] KurokoBK registered 177.37.143.2
[08-27 01:50:44]: [FINE] KurokoBK logged in 177.37.143.2
[08-27 03:01:06]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[08-27 18:59:29]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[08-27 18:59:31]: [INFO] MySQL setup finished
[08-27 18:59:31]: [INFO] No supported permissions system found! Permissions are disabled!
[08-27 18:59:32]: [WARN] WARNING! The protectInventory feature requires ProtocolLib! Disabling it...
[08-27 18:59:32]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[08-27 18:59:39]: [INFO] Downloading GEO IP database, because the old database is older than 30 days or doesn't exist
[08-27 18:59:39]: [WARN] No MaxMind credentials found in the configuration file! GeoIp protections will be disabled.
[08-27 18:59:39]: [INFO] There is no newer GEO IP database uploaded to MaxMind. Using the old one for now.
[08-27 18:59:39]: [WARN] Could not download GeoLiteAPI database [FileNotFoundException]: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
[08-27 18:59:39]: java.io.FileNotFoundException: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
	at java.base/java.io.RandomAccessFile.open0(Native Method)
	at java.base/java.io.RandomAccessFile.open(RandomAccessFile.java:356)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:273)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:223)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.BufferHolder.<init>(BufferHolder.java:20)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.Reader.<init>(Reader.java:121)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.startReading(GeoIpService.java:179)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.updateDatabase(GeoIpService.java:150)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftTask.run(CraftTask.java:101)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[08-27 19:00:00]: [FINE] TemSaborDeMel logged in 177.51.0.200
[08-27 19:42:04]: [FINE] TemSaborDeMel logged in 177.51.0.200
[08-28 03:01:06]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
[08-29 17:26:38]: [INFO] Connection arguments loaded, Hikari ConnectionPool ready!
[08-29 17:26:41]: [INFO] MySQL setup finished
[08-29 17:26:41]: [INFO] No supported permissions system found! Permissions are disabled!
[08-29 17:26:42]: [WARN] WARNING! The protectInventory feature requires ProtocolLib! Disabling it...
[08-29 17:26:42]: [INFO] AuthMe 5.6.0 build n.CUSTOM successfully enabled!
[08-29 17:26:49]: [INFO] Downloading GEO IP database, because the old database is older than 30 days or doesn't exist
[08-29 17:26:49]: [WARN] No MaxMind credentials found in the configuration file! GeoIp protections will be disabled.
[08-29 17:26:49]: [INFO] There is no newer GEO IP database uploaded to MaxMind. Using the old one for now.
[08-29 17:26:49]: [WARN] Could not download GeoLiteAPI database [FileNotFoundException]: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
[08-29 17:26:49]: java.io.FileNotFoundException: plugins/AuthMe/GeoLite2-Country.mmdb (No such file or directory)
	at java.base/java.io.RandomAccessFile.open0(Native Method)
	at java.base/java.io.RandomAccessFile.open(RandomAccessFile.java:356)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:273)
	at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:223)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.BufferHolder.<init>(BufferHolder.java:20)
	at AuthMe-5.6.0.jar//fr.xephi.authme.libs.com.maxmind.db.Reader.<init>(Reader.java:121)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.startReading(GeoIpService.java:179)
	at AuthMe-5.6.0.jar//fr.xephi.authme.service.GeoIpService.updateDatabase(GeoIpService.java:150)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftTask.run(CraftTask.java:101)
	at org.bukkit.craftbukkit.v1_19_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)

[08-29 17:29:08]: [FINE] TemSaborDeMel logged in 181.77.99.58
[08-29 17:29:13]: [WARN] Player 'Manazeus364' wasn't online during login process, aborted...
[08-29 17:29:13]: [FINE] Manazeus364 registered 177.51.3.245
[08-29 17:29:52]: [FINE] Manazeus364 logged in 177.51.3.245
[08-30 03:01:06]: [INFO] AuthMe 5.6.0-bCUSTOM disabled!
